### DISCUSSION TITLE
how would I get the external IP the VPN is using

### DISCUSSION THREAD

**Discussion by Solamain** [bubuntux/nordlynx#112] [Q&A]
Title pretty much says it all.

**Comment by noelhibbard** [comment]
Something simple like this would work:
```docker exec -t <container_name> curl ip.me```

**Comment by instantdreams** [comment]
An external check would be to set up a torrent client to use your nordlynx network, and follow these steps:
  1. Open a browser to https://torguard.net/checkmytorrentipaddress.php
  2. Copy the Magnet URI link
  3. Add the URL link to your torrent client
  4. Wait a minutes and check the ip address on the tracking page
  5. Compare to the assigned ip address on your router
